lesson05/assignment/css/_units.scss

```diff
@@ -0,0 +1,61 @@
+// Fonts
+$xs-font: 1rem;
+$s-font: 1.5rem;
+$m-font: 2rem;
+$l-font: 3rem;
+$xl-font: 5rem;
+
+// Pixels
+$xl-pxl: 10px; 
+$l-pxl: 8px;
+$m-pxl: 6px;
+$s-pxl: 4px;
+$xs-pxl: 3px;
+$xxs-pxl: 2px;
+$xxxs-pxl: 1px;
+
+// Rems
+$micro-pos-value: .10rem; 
+$micro-neg-value: -0.10rem;
+$xxxxs-rem: .2rem;
+$xxxs-rem: .25rem;

```

Comment:
These names would be easy to misspell. Maybe a different naming convention would be better for the very small ones? Something like:
```
$tiniest-rem: .2rem;
$tinier-rem: .25rem;
$tiny-rem: .5rem;
$xs-rem: 1rem;
```